### PR TITLE
Docs for #363

### PR DESCRIPTION
**Pull request by LiraLemur**
Signed-off-by: Sara <lira.lemur@gmail.com>

### Description of the Change

Added a few lines of config docs for #363
Changed an example config as well (added this parameter)

### Benefits

Better docs

### Possible Drawbacks 

 ¯\_(ツ)_/¯

### CODE REVIEW

File: docs/source/guides/configuration.rst [hunk 1]
```diff
@@ -27,6 +27,7 @@ at ``example/config.sample``
     "vote_delay": 5000,
     "mst_enable" : false,
     "mst_expiration_time" : 1440,
+    "mst_stalled_batch_threshold" : 1800000,
```
Comment (MBoldyrev) [inline docs/source/guides/configuration.rst]:
```suggestion
    "mst_stale_threshold_milliseconds" : 1800000,
```
the related file is https://github.com/hyperledger/iroha/blob/ee6b906daade545019e331f00cc7b68a81507d98/irohad/main/iroha_conf_literals.cpp#L26

please consider other places too

File: docs/source/guides/configuration.rst [hunk 2]
```diff
@@ -101,6 +102,8 @@ Environment-specific parameters
   This parameter allows users to find an optimal value in a tradeoff between
   resource consumption and the delay of getting back to work after an idle
   period.
+- ``mst_stalled_batch_threshold`` is an optional parameter that sets the time threshold after which inactive batches are considered not present on other peers (in milliseconds).
```
Comment (MBoldyrev) [inline docs/source/guides/configuration.rst]: Let's bind this abstract statement to user's needs:
- if a node was restarted, it will recover MST state after approx this period
- in case of network failures, same
- if too small, odd traffic is generated and processor gets busy
- if too large, recovery from MST problems delays

File: docs/source/guides/configuration.rst (hunk 1 above)
Comment (LiraLemur) [reply docs/source/guides/configuration.rst]: Thanks! please check if everything is ok now!

File: docs/source/guides/configuration.rst [hunk 3]
```diff
@@ -101,6 +102,10 @@ Environment-specific parameters
   This parameter allows users to find an optimal value in a tradeoff between
   resource consumption and the delay of getting back to work after an idle
   period.
+- ``mst_stale_threshold_milliseconds`` is an optional parameter that sets the time after which the node that sent the MST transaction in the first place would try to send it again if no final status is received.
+  It is needed in case, say, some nodes failed and restarted or experienced network problems, so could not receive the transaction.
+  Default value is 1800000 (30 minutes).
+  We recommend not to set it to too small value – it will be using processing resources constantly connecting the nodes and also not to set it to too big value – that will cause delays in receiving MST transactions in case of failed nodes.
```
Comment (MBoldyrev) [inline docs/source/guides/configuration.rst]: > <...> of failed nodes

this also concerns failed network 

maybe rephrase "too (small/big)" with something more neutral, like "increasing this value leads to..."?
